src/_sass/docs.scss

```diff
@@ -164,6 +168,10 @@ $sidebar-maximum-width: 400px;
 // --------------------------------------------------
 .Docs-sidebar {
   background-color: $color-white;
+  border-right: 2px solid $tilt2-color-gray-lightest;
+  position: sticky;
+  top: 64px;
```

Comment:
When I tested, this gap at the top of the sidebar is just a bit shorter than the height of the sticky horizontal menu with Docs categories. We should make these match up so the overall effect is more refined.

In the past, I've done this by using a shared variable to set the height of the menu _and_ the sidebar-top-gap.